.github/workflows/test-s3gw.yml

```diff
@@ -8,82 +8,106 @@ on:
-        id: ccache_timestamp
-        run: |
-          echo "timestamp=$(date +%Y-%m-%d)" >> $GITHUB_OUTPUT
-
-      - name: Cache CCache Files
-        uses: actions/cache@v3.0.4
+      - name: Checkout s3gw
+        uses: actions/checkout@v3
         with:
-          path: ceph/build.ccache
-          key: ccache-${{ steps.ccache_timestamp.outputs.timestamp }}
-          restore-keys: |
-            ccache-
-
-      - name: Build radosgw binary and tests
-        run: |
-          docker run --rm \
-            -v $GITHUB_WORKSPACE/ceph:/srv/ceph \
-            -e NPROC=16 \
-            -e WITH_TESTS=ON \
-            quay.io/s3gw/build-radosgw:latest
+          repository: aquarist-labs/s3gw
+          path: s3gw
+          submodules: false
 
-      - name: Pack Build artifacts
-        run: |
-          tar -czvf build-results.tar.gz \
-            ceph/build/bin/radosgw \
-            ceph/build/lib/libceph-common.so \
-            ceph/build/lib/libceph-common.so.2 \
-            ceph/build/lib/librados.so \
-            ceph/build/lib/librados.so.2 \
-            ceph/build/lib/librados.so.2.0.0
-
-      - name: Save radosgw binary
-        uses: actions/upload-artifact@v3
+      - name: Checkout s3tests
+        uses: actions/checkout@v3
         with:
-          name: radosgw-${{ steps.ccache_timestamp.outputs.timestamp }}
-          path: build-results.tar.gz
+          repository: ceph/s3-tests
+          path: s3tests
 
-      - name: Run SFS unit tests
-        run: |
-          docker run \
-            -v $GITHUB_WORKSPACE/ceph/build/bin:/radosgw/bin \
-            -v $GITHUB_WORKSPACE/ceph/build/lib:/radosgw/lib \
-            quay.io/s3gw/run-radosgw-tests
+      - name: Checkout s3gw-status
+        uses: actions/checkout@v3
+        with:
+          repository: ${{ github.repository_owner }}/s3gw-status
+          path: s3gw-status
 
-      - name: Install S3cmd
+      - name: Configure Git
         run: |
-          sudo apt-get update
-          sudo apt-get install -y s3cmd
+          git config --global user.name "GitHub Actions Bot"
+          git config --global user.email "actions-bot@users.noreply.github.com"
 
       - name: Setup Python
         uses: actions/setup-python@v4
         with:
           python-version: 3.8
 
-      - name: Install Python Dependencies
+      - name: Install Dependencies
         run: |
+          YQ_GH_URL=https://github.com/mikefarah/yq/releases/download
+          YQ_VERSION=v4.31.1
+          YQ_CHECKSUM=1aef844cbecbbf81036449ea5e7dfcdf19d2d374fab6303fdb8f849d04275d76
+
+          sudo apt-get update
+          sudo apt-get install -y \
+            bc \
+            wget \
+            s3cmd
+
+          # Unfortunately, since yq is only available through snap on Ubuntu and
+          # that doesn't work in docker containers (at least not out of the
+          # box), this abomination is the way to go to install yq.
+          echo "${YQ_CHECKSUM}  yq" >> checksum
```

Comment:
nit/optional: bash fd redirection saves the temp file: sha256sum -c <(echo "${YQ_CHECKSUM}  yq")